weekly-updates/weekly-update.2015-05-08.md

```diff
@@ -0,0 +1,78 @@
+# io.js 2.0 releases
+This week we had two io.js releases [v2.0.0](https://iojs.org/dist/v2.0.0/) and [v2.0.1](https://iojs.org/dist/v2.0.1/), complete changelog can be found [on GitHub](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md).
+
+### Notable changes
```

Comment:
IMO, we should not merge changelog v2.0.0 and v2.0.1. We should write the changes on each version. 
like: https://github.com/iojs/evangelism/blob/master/weekly-updates/weekly-update.2015-03-20.md